### PR TITLE
Review of docs for grammar, etc.

### LINKED ISSUE #36: Proof-reading of documentation
As pointed out by @TomHAnderson in #34, the documentation currently contains a number of grammatical errors. These should be fixes before publishing the documentation on www.doctrine-project.org. Tom volunteered to do this.

### PR DESCRIPTION
**Pull request by TomHAnderson**
Many of the docs were written in the first person:  "I recommend" and "you should".  This familiarity of the reader is not technical in its approach and such language is removed in this PR.

This PR addresses the text of the documentation and does not review the code examples.

### CODE REVIEW

File: docs/en/performance-considerations.rst [hunk 1]
```diff
@@ -231,41 +227,41 @@ action will look something like this:
...
-happens during the extracting phase. Here is how it works : when an
-object is bound to the form, this latter iterates through all its
-fields, and tries to extract the data from the object that is bound. In
-our example, here is how it works:
-
-1. It first arrives to the UserFieldset. The input are “name” (which is
-   string field), and a “city” which is another fieldset (in our User
+This looks good, yes? But, a check for the queries that are
+made (for instance using the
+`Laminas:raw-latex:`\DeveloperTools `module <https://github.com/laminas/laminas-developer-tools>`__
```
Comment (driehle) [inline docs/en/performance-considerations.rst]: Shouldn't this be 

```
`Laminas\\DeveloperTools module <...>`
```
?

**Review by driehle — APPROVED** [review]

File: docs/en/performance-considerations.rst (hunk 1 above)
Comment (TomHAnderson) [reply docs/en/performance-considerations.rst]: Fixed.  Maybe this was an artifact from the conversion?

Comment (driehle) [reply docs/en/performance-considerations.rst]: Yes, I think so

**Comment by driehle** [comment]
LGTM :ship:

**Pull request merged**